resources/kcp/charts/kyma-environment-broker/templates/deployment.yaml

```
@@ -424,18 +424,20 @@ spec:
           securityContext:
 {{ toYaml . | indent 12 }}
           {{- end }}
-         {{end}}
+       {{end}}
```

nit: consider adding `-`, that way it shouldn't render extra empty line
```suggestion
       {{- end}}
```